### PR TITLE
Validation fixes

### PR DESCRIPTION
**Pull request by chrisms86**
## Category

- [x ] validation fix

## What's in this Pull Request?

validation fixes

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[bd04d8b](https://github.com/chrisms86/sp-dev-docs/commits/bd04d8bdf7dc5d0338ae1b3e0a0e349dcd48e94e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenario-guidance/Security.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/scenario-guidance/Security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/scenario-guidance/security?branch=pr-en-us-8028) |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206061323313621-8028/BuildReport?accessString=f92035e54bd6cf63c05f4cba5b381fbac1e56c297293b2b8adad6dd6510b296a).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/scenario-guidance/Security.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Security
+description: Outlines how SharePoint supports security for user access at the website, list, list or library folder, and item levels.
```
Comment (MeeraDi) [inline docs/scenario-guidance/Security.md:3]: It seems like this article is providing links to other articles.

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[b56933f](https://github.com/chrisms86/sp-dev-docs/commits/b56933f7b214f3f6d65e5610bf87e19eeac9ff50)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenario-guidance/Security.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/scenario-guidance/Security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/scenario-guidance/security?branch=pr-en-us-8028) |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206061517158860-8028/BuildReport?accessString=4490e6b870dc078fd7663eec8f3dbe6e4ea9682c4693246f84889e49522d8a84).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by andrewconnell — CHANGES_REQUESTED** [review]
Please update the `ms.date` with the current date of the change to correctly reflect the last time this doc was updated.

**Review by andrewconnell — APPROVED** [review]

**Comment by chrisms86** [comment]
ms.date updated

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[e4d8819](https://github.com/chrisms86/sp-dev-docs/commits/e4d8819d2ce6c4d3f1c8de5fdabfc10eccc87c7a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenario-guidance/Security.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/scenario-guidance/Security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/scenario-guidance/security?branch=pr-en-us-8028) |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206071422599663-8028/BuildReport?accessString=d269f895238ed331678d62b9a70f390de3a309154437de9d73e404927f590694).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**